src/exploratory.py

```diff
@@ -883,7 +891,7 @@ def estimate_ltv_impact(
         """
         print(output_txt)
 
-        return data
+        return data.round(self.decimals)
```

Comment:
Is there any case where a user doesn't want to have any rounding? A work around can be to add a large enough number as input to the param, but another solution could be to allow null and don't apply rounding. (only if there is a use case for it)